src/controls.css

```diff
@@ -260,6 +260,7 @@ input[type="checkbox"][role="switch"]:checked::after {
 select {
     padding-top: 0.6rem;
     padding-bottom: 0.6rem;
+    padding-right: 1.5rem;
```

Comment:
I'd use `padding-inline-end` if I were you instead of `padding-right`, that way if `body` has `dir="rtl"` then it works okay for right-to-left langs too

Reply:
And you're right in using `padding` instead of `margin` in the issue suggestion too lol